### PR TITLE
GDPR-103 Updating to use new DPS CircleCI Orb

### PR DESCRIPTION
**Pull request by brightonsbox**

### CODE REVIEW

File: helm_deploy/values-dev.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 replicaCount: 2
 
 image:
-  repository: docker.io/mojdigitalstudio/dps-data-compliance
+  repository: quay.io/hmpps/dps-data-compliance
```
Comment (andymarke) [inline helm_deploy/values-dev.yaml:7]: I wonder why we have these in each values file when they could be variables, I noticed other projects are the same ...

File: docker-compose.yml [hunk 2]
```diff
@@ -71,7 +71,7 @@ services:
       - AWS_SECRET_ACCESS_KEY=arbitrary
 
   pathfinder-api:
-    image: mojdigitalstudio/pathfinder-api:latest
+    image: quay.io/hmpps/pathfinder-api:latest
```
Comment (andymarke) [inline docker-compose.yml:74]: I assume this has been migrated to quay? It is difficult to keep track

File: helm_deploy/values-dev.yaml (hunk 1 above)
Comment (brightonsbox) [reply helm_deploy/values-dev.yaml:7]: Yes that's true! I'll have a look at that in a follow up.

File: docker-compose.yml (hunk 2 above)
Comment (brightonsbox) [reply docker-compose.yml:74]: I think the pathfinder-api (as opposed to the main pathfinder project) started its life in quay

Comment (brightonsbox) [reply docker-compose.yml:74]: So this is a correction really, rather than an update

**Pull request merged**